### PR TITLE
COMMERCE-7000 Escape admin breadcrumb url

### PR DESCRIPTION
**Pull request by jaydawu**
https://liferay.atlassian.net/browse/COMMERCE-7000
Resent from https://github.com/liferay-commerce/liferay-portal/pull/4035

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceTaxEngine.testcase [hunk 1]
```diff
@@ -726,6 +726,40 @@ definition {
 			value1 = "46.00");
 	}
 
+	@description = "This is a test for COMMERCE-7000. When the user updates a tax categories entry's details URL redirect part to '%22%3E%3Cscript%3Ealert(111)%3C/script%3E' and visits the link, then no XSS is present"
+	@priority = 3
+	test AssertNoXSSWithTaxCategoriesURL {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceTaxEngine.testcase:731]:
```suggestion
	test CannotExecuteXSSInTaxCategoriesEntryURL {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceTaxEngine.testcase [hunk 2]
```diff
@@ -726,6 +726,40 @@ definition {
...
+		property portal.acceptance = "false";
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceTaxEngine.testcase:732]: As this is a security bug, please include the test in the relevant

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Review by lorenzolapescara — APPROVED** [review]

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:forward

**Pull request closed without merging**